docs/settings/alert-action-settings.asciidoc

```diff
@@ -207,3 +207,16 @@ Specifies the behavior when a new or changed rule has a schedule interval less t
 
 `xpack.alerting.rules.execution.actions.max`::
 Specifies the maximum number of actions that a rule can trigger each time detection checks run.
+
```

Comment:
Related to https://github.com/elastic/kibana/pull/128294/files#r833802664, I think we should change the "xpack.alerting.rules.execution.actions.max" setting to avoid the use of "execution" terminology. For example, can we just use "xpack.alerting.rules.actions.max"? The description makes it clear that it's a "per check" max, so I don't think losing "execution" hurts us here. If we must keep another noun, "invocation" or "check" or "event" are less problematic.

Reply:
Hi Lisa, thank you for the alternatives :) 
We have created a follow up issue to rename `execution`
https://github.com/elastic/kibana/issues/128512
Would like to merge these PR's and fix it with the follow up issue, v8.2 will not be released with that terminology.

Reply:
Sounds good, thanks!